src/move/move.ts

```diff
@@ -43,6 +43,14 @@ export const checkUserMove = (
     };
   }
 
+  if (action.value === "log") {
+    console.log(board);
+    return {
+      gameState: { ...gameState },
+      allowedMove: false,
+    };
+  }
```

Comment:
This is both very odd and yet very clever IMO :joy: .
I don't know what to think about this... Let's keep it! :grin: